tests/tests_unit/graph/test_transformers/test_prune_graph.py

```diff
@@ -185,22 +340,24 @@ def test_two_hop_flattener_keep_connecting_node(
         target_node_type: URIRef,
         namespace: Namespace,
         target_property: str,
-        target_property_holding_new_property_name: str | None,
+        target_property_holding_new_property: str | None,
         expected_triples: list[Triple],
+        convert_literal_to_uri: bool,
```

Comment:
Not necessary, but I think these tests can be further simplified by creating the `AttachPropertyFromTargetToSource` transformer in the test cases, and pass the entire object instead of all the individual parameters. Typically, you want to avoid doing expensive operations in the construction of the test cases, but since there is no logic in the `__init__` of `AttachPropertyFromTargetToSource` this is not a problem.

```python
def test_two_hop_flattener_keep_connecting_node(
        self,
        triples: list[Triple],
        transformer: AttachPropertyFromTargetToSource,
        expected_triples: list[Triple],
    ) -> None:
 ....

Reply:
Agree , and there are a few more improvements such as better bundling of initial triples and expected.